src/components/MessagesList/MessagesList.vue

```diff
@@ -292,6 +299,10 @@ export default {
 			}
 			return (this.conversation.unreadMessages >= summaryThreshold)
 		},
+
+		hasPinnedMessages() {
+			return this.conversation?.lastPinnedId && this.conversation?.lastPinnedId !== this.conversation?.hiddenPinnedId
```

Comment:
This should certainly be there in this case
```suggestion
			return this.conversation?.lastPinnedId && this.conversation.lastPinnedId !== this.conversation.hiddenPinnedId
```